EIPS/eip-1.md

```diff
@@ -42,21 +42,27 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
 
 Standards Track EIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification]. The EIP should be reviewed and accepted before an implementation is begun, unless an implementation will aid people in studying the EIP. Standards Track Core EIPs must be implemented in at least three viable Ethereum clients before it can be considered Final.
 
-For an EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+Standards track EIPs of type 'Core' are updated from 'Draft' to 'Accepted' status when a core devs meeting has accepted the draft and the participants have announced their intention to implement the EIP for inclusion in a future hard fork. Once a Core EIP has been implemented in at least two clients totalling at least 50% of deployed nodes, and all pass a common set of test suites, the status of the EIP is updated to 'Final'. An EIP can also be assigned status “Deferred”. The EIP author or editor can assign the EIP this status when no progress is being made on the EIP.
+
+Non-core EIPs remain in Draft status until at least one implementation has been completed and referenced in the Implementations section of the draft. At that point, the draft's author may request a 'final call', by submitting a pull request changing the EIP's status from 'Draft' to 'Final'.
+
+Final call PRs will be left open for two weeks to give community members an opportunity to comment. An editor will review the feedback on the final call; if there are no significant unaddressed technical issues, the PR will be merged and the EIP transitioned to 'Final'. If unaddressed technical issues remain, the PR will be closed, and the author invited to resubmit when the EIP is more mature.
 
-Once an EIP has been accepted, the implementations must be completed. When the implementation is complete and accepted by the community, the status will be changed to “Final”.
+Editors may close a final-call PR without waiting for the two week period to expire if the EIP is not clear and unambiguous, has significant formatting or descriptive issues, or if the feedback already submitted by the community makes it clear the EIP is not ready to be finalised.
 
-An EIP can also be assigned status “Deferred”. The EIP author or editor can assign the EIP this status when no progress is being made on the EIP. Once an EIP is deferred, the EIP editor can re-assign it to draft status.
+Once an EIP is marked Final, edits are not permitted unless they correct errata; no substantive changes may be made. Anyone wishing to amend a Final EIP must submit a new EIP instead.
 
 An EIP can also be “Rejected”. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
```

Comment:
In addition to "Rejected" there are the following set of statuses most of which overlap in meaning: 
- Deferred
- Withdrawn
- Superceded
- Replaced
- Closed
Can we shrink this list and provide guidance for each?